deployment/aws/ec2/data-prepper-ec2-deployment-cfn.yaml

```diff
@@ -144,6 +158,8 @@ Resources:
         # Script to download and run Data Prepper
         Fn::Base64: !Sub |
           #!/bin/bash
+          export JAVA_OPTS="-xms${HeapInMb}m -xmx${HeapInMb}m"
```

Comment:
The first x's need to be capitalized: `-Xms256m -Xmx256m` (ref https://docs.oracle.com/cd/E15523_01/web.1111/e13814/jvm_tuning.htm#PERFM161)

Reply:
Lol That was dumb :(

Reply:
No problem, you found a bug! Created issue #313

Reply:
I actually added the fix to append error to the same output file.